packages/firestore/test/integration/util/helpers.ts

```diff
@@ -255,6 +255,14 @@ export function withTestCollection(
   return withTestCollectionSettings(persistence, DEFAULT_SETTINGS, docs, fn);
 }
 
+export function withEmptyTestCollection(
```

Comment:
Can `withEmptyTestCollection()` be written in terms of `withTestCollection()` like this:

```
return withTestCollection(persistence, {}, fn);
```